CONTRIBUTORS

```diff
@@ -12,6 +12,7 @@ This project was previously released under the MIT license, but it has since bee
 
 ## List of Contributors
 
-* 
+- [@Hyask](https://github.com/Hyask) — Florent "Skia" Jacquet <skia@libskia.so>
```

Comment:
My current email address is actually `skia@hya.sk`. `libskia.so` doesn't exist anymore.